plugin-code/src/main/resources/template/css/cluecumber.css

```diff
@@ -231,4 +231,13 @@ a.anchor {
     position: relative;
     top: -2.5rem;
     visibility: hidden;
+}
+
+parameter {
```

Comment:
You are introducing two custom (non-standard) html tags here instead of using css classes or data attributes. This should not be done.

Reply:
Other than that I like the approach.

Reply:
why you do not like custom tags? I will change them in class but wonder why you are not in favor of such practice?

Reply:
Although browsers support creating a custom element, it's not officially supported by the HTML5 standard. So this technically breaks the standard.